src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,21 @@
 import React from 'react';
+
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
+  <>
+    <div className="movies">
+      {movies.map(movie => (
+        <div className="card" data-cy="Movie" key={movie.imdbId}>
+          <MovieCard {...movie} />
```

Comment:
you don't need to do destructuring here, let's pass movie and do destructuring  in MovieCard